packages/components/src/components/switch/switch.stories.ts

```diff
@@ -1,16 +1,20 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
+
 const { argTypes, parameters } = storybookDefaults('sd-switch');
-const { generateTemplate } = storybookTemplate('sd-switch');
 const { overrideArgs } = storybookHelpers('sd-switch');
+const { generateTemplate } = storybookTemplate('sd-switch');
+
+/**
+ * Gives control over a feature or option that can be turned on or off.
```

Comment:
```suggestion
 * Used to give control over a feature or option that can be turned on or off.
```

<img width="630" alt="image" src="https://github.com/user-attachments/assets/2e0435d0-2f64-4c2a-96bd-d04dd6ded159">